main.go

```diff
@@ -72,8 +73,48 @@ const (
 	AccountEmpty
 )
 
+// Result is the action taken by a swipe of a card. It contains all information
+// to be communicated to the user.
+type Result struct {
+	Code    ResultCode
+	UID     []byte
+	User    string
+	Account float32
+}
+
+func flashLCD(lcd *lcd2usb.Device, text string, r, g, b uint8) error {
+	lcd.Color(r, g, b)
+	for i, l := range strings.Split(text, "\n") {
```

Comment:
The text could easily be more than 16 characters. This will probably mess up the display.